processor/transformprocessor/README.md

```diff
@@ -377,6 +379,24 @@ Examples:
 
 - `convert_summary_sum_val_to_sum("cumulative", false)`
 
+### convert_summary_quantile_val_to_gauge
+
+`convert_summary_quantile_val_to_gauge(Optional[attributeKey])`
+
+The `convert_summary_quantile_val_to_gauge` function creates a new Gauge metric and injects each of the Summary's quantiles into a single Gauge datapoint.
+
+`attributeKey` is an optional string that specifies the attribute key holding the quantile value for each corresponding output data point. The default key is `quantile`.
+
+The name for the new metric will be `<summary metric name>`. The fields that are copied are: `timestamp`, `starttimestamp`, `attributes`, `unit` and `description`. The new metric that is created will be passed to all functions in the metrics statements list.  Function conditions will apply.
```

Comment:
I'm a bit concerned that after running this function, users will have two differently-typed metrics with the same name. This won't be too much of an issue since users can still select on them using the name and metric type, but it's likely they will either want to rename one of the metrics or delete the old metric.

Should we:
1. Document this and tell users they need to resolve the conflicting names themselves.
2. Add a parameter that resolves this automatically.

Reply:
added the optional suffix